docker-run.sh

```diff
@@ -0,0 +1,17 @@
+#/bin/bash
+
+docker run \
+  -e "SLACK_BOT_TOKEN=" \
+  -e "AWS_ACCESS_KEY_ID=" \
+  -e "AWS_SECRET_ACCESS_KEY=" \
+  -e "GDOCS_SERVICE_ACCOUNT_FILENAME=/gdocs-service-account.json" \
```

Comment:
We should probably refactor this to post results elsewhere, but it's probably out of scope for this ticket. I'm not sure who owns the Google account this currently goes to, but I'd assume it's Stephen's.

Reply:
Stephen transferred the ownership of the google doc itself; I don't have any idea how this service account works though or how it is created. But I agree, anything Seeq + Google Account is really sketchy though.